FFXIV_Modding_Tool/Validation.cs

```diff
@@ -80,8 +80,14 @@ public bool ValidateBackups()
                     }
                 }
             }
-            if (problemFound)
+            if (problemFound){
+                if (PromptContinuation("Would you like to back up now?")){
+                    BackupIndexes();
```

Comment:
BackupIndexes() is a function that comes from main, to access the function this should be main.BackupIndexes();